src/operun/crm/theme/js/crm.js

```diff
@@ -1,6 +1,12 @@
 (function($) {
   "use strict"; // Start of use strict
 
-  $(document).ready(function() {});
+  $(document).ready(function() {
+    $('#form-widgets-title').change(function() {
```

Comment:
Please check if firstname and lastname is empty bevor fill in. Please restrict to content types Account and Contact.